File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        vector<int> nums_doubled(2*n);
        for(int i = 0; i < n; i++){
            nums_doubled[i]= nums[i];
            nums_doubled[i+n] = nums[i];
        }
    
    return nums_doubled;
    }
};

File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        swap_val = 0
        for i in range(0,len(nums)-1):
            for j in range(i+1, len(nums)):
                if(nums[i]>nums[j]):
                    swap_val = nums[j]
                    nums[j] = nums[i]
                    nums[i] = swap_val
                